src/Orchard.Tests.Modules/Migrations/SchemaCommandGeneratorTests.cs

```diff
@@ -118,6 +118,7 @@ public class SchemaCommandGeneratorTests {
         }
 
         public class StubFolders : IExtensionFolders {
+            public IEnumerable<string> Paths { get { return AvailableExtensions().Select(e => e.Location).Distinct(); } }

```

Comment:
This could go into an abstract base class instead of being copied over multiple times.